client/layout/masterbar/logged-in.jsx

```diff
@@ -556,6 +570,9 @@ export default connect(
 			currentLayoutFocus: getCurrentLayoutFocus( state ),
 			hasDismissedThePopover: getPreference( state, MENU_POPOVER_PREFERENCE_KEY ),
 			isFetchingPrefs: isFetchingPreferences( state ),
+			/** If the user is newer than new navigation shipping date, don't tell them this nav is new. Everything is new to them */
```

Comment:
Nit: for one-line comments we usually use `//` rather than `/* */`. This also makes the line shorter.
```suggestion
			// If the user is newer than new navigation shipping date, don't tell them this nav is new. Everything is new to them
```